paper.qmd

```diff
@@ -310,6 +310,9 @@ cross-platform distribution is well-established. Indeed, the two approaches are
 not mutually exclusive. As shall be explained in @sec-nix, it is possible to use
```

Comment:
It is also possible to build docker containers (or VM images) with nix (or Guix). So you don't have to use docker's build tooling or a Dockerfile at all you can build containers with an entirely nix toolchain and push the resulting images to container registries. This is probably a slightly more reproducible way of building containers than the docker native way.

Though running a nix environment in a container of another linux image can be a simpler way to use nix with existing CI tooling.

https://nix.dev/tutorials/nixos/building-and-running-docker-images.html